src/components/common/ContactInfo.vue

```diff
@@ -18,12 +18,7 @@
   </ul>
 </template>
 
-<script lang="ts">
-import Vue from 'vue'
-import { Component } from 'vue-property-decorator'
-
-@Component({})
-export default class ContactInfo extends Vue {}
```

Comment:
How does this component get named?

Reply:
The component is named as a variable when it is imported.

i.e. 
```import ContactInfo from './ContactInfo.vue
<ContactInfo />```  works for this component but 

``` import SomeRandomName from './ContactInfo.vue'
<SomeRandomName />``` works exactly the same

Reply:
So then we can't do `import * from <whatever>` ?

Reply:
I think you do would do ```import * as All from ./components <All.Foo /> <All.Bar />``` There may be a more elegant solution but I haven't found one yet

Reply:
Ok